### DISCUSSION TITLE
JWS-CT

### DISCUSSION THREAD

**Discussion by frosty5689** [panva/jose#233] [Ideas]
Hello,

I recently started using this library for JWS-CT which has been really helpful. So first of all thank you for your hardwork on maintaining this project! I'm wondering if there are plans to extend CompactSign and CompactVerify to potentially support the workflow described in JWS-CT. Specifically the latest draft https://tools.ietf.org/id/draft-jordan-jws-ct-02.html

I get it is just another use case within JWS spec, so it will mostly just be a helper class that takes the mangling of the resulting Compact JWS out of the hands of the application developer.

Thanks!

**Comment by panva** [comment]
Hello @frosty5689 

I have no immediate plans to support this individual draft with a dedicated API. 

Should that draft become an IETF WG item and eventually an RFC I will revisit it.

**Comment by panva** [comment]
I think it would be immensely helpful for others who stumble here if you provided a snippet of code about "how" you use jose to issue tokens following that draft though.
